tools/exports.js

```diff
@@ -17,3 +17,4 @@ exports["string_template"] = string_template;
 exports["tokenizer"] = tokenizer;
 exports["is_identifier"] = is_identifier;
 exports["SymbolDef"] = SymbolDef;
+exports["EXPECT_DIRECTIVE"] = EXPECT_DIRECTIVE;
```

Comment:
A simple change to the code generator in tools/node.js should be enough to pass a DEBUG flag.